gnomad_qc/analyses/compute_pext.py

```diff
@@ -0,0 +1,279 @@
+    per_sample_pipeline.add_steps(
+        {
+            "annotation_level": annotation_level,
+            "base_level": base_level,
+            "browser_ht": browser_ht,
+        }
+    )
+
+    return per_sample_pipeline
+
+
+def main(args):
+    """Script to get release pext annotation files."""
+    test = args.test
+    gtex_version = args.gtex_version
+    vep_version = args.vep_version
+    overwrite = args.overwrite
+
+    hl.init(
+        log="/pext.log",
+        default_reference="GRCh37" if gtex_version == "v7" else "GRCh38",
+        tmp_dir="gs://gnomad-tmp-4day",
+    )
+
+    pext_res = get_pipeline_resources(test, gtex_version, vep_version, overwrite)
+    ht = pext_res.context_ht.ht().naive_coalesce(5000)
+    tx_mt = pext_res.tx_mt.mt()
+
+    if test:
+        ht, tx_mt = filter_to_test(ht, tx_mt)
+
+    logger.info(
+        "Summarizing transcript expression for GTEx version %s...",
+        gtex_version,
+    )
+    tx_ht = summarize_transcript_expression(tx_mt).checkpoint(
+        new_temp_file(f"{f'test_' if test else ''}tx_ht", "ht"), overwrite=overwrite
+    )
+
+    if args.annotation_level:
+        logger.info("Creating annotation-level pext scores...")
+        res = pext_res.annotation_level
+        res.check_resource_existence()
+
+        perform_tx_annotation_pipeline(
+            ht, tx_ht, tissues_to_exclude_from_mean=TISSUES_TO_EXCLUDE[gtex_version]
+        ).write(res.annotation_pext_ht.path, overwrite=overwrite)
+
+    if args.base_level:
+        logger.info("Creating base-level pext scores...")
+        res = pext_res.base_level
+        res.check_resource_existence()
+
+        perform_tx_annotation_pipeline(
+            ht,
+            tx_ht,
+            additional_group_by=["gene_symbol"],
+            tissues_to_exclude_from_mean=TISSUES_TO_EXCLUDE[gtex_version],
+        ).write(res.base_pext_ht.path, overwrite=overwrite)
+
+    if args.browser_ht:
+        logger.info("Creating pext Table for browser loading...")
+        res = pext_res.browser_ht
+        res.check_resource_existence()
+        base_pext_ht = res.base_pext_ht.ht()
+
+        tissue_map = {
+            t: clean_tissue_name_for_browser(t) for t in hl.eval(base_pext_ht.tissues)
+        }
+        base_pext_ht = base_pext_ht.annotate(
+            **{k: base_pext_ht[k].expression_proportion for k in tissue_map}
+        )
+        base_pext_ht = base_pext_ht.rename(tissue_map)
+        base_pext_ht = base_pext_ht.annotate_globals(
+            tissues=base_pext_ht.tissues.map(lambda x: hl.literal(tissue_map).get(x))
+        )
+        create_tx_annotation_by_region(base_pext_ht).write(
+            res.browser_pext_ht.path, overwrite=overwrite
+        )
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description="This script creates the pext annotation files for release."
+    )
+    parser.add_argument("--overwrite", help="Overwrite data", action="store_true")
```

Comment:
```suggestion
    parser.add_argument("--overwrite", help="Overwrite data.", action="store_true")
```
nit for consistency with the other descriptions